web/test/pages/roadmap.test.tsx

```diff
@@ -409,4 +410,46 @@ describe("roadmap page", () => {
     expect(within(sectionStart).getByText("step2")).toBeVisible();
     expect(within(sectionPlan).getByText("step1")).toBeVisible();
   });
+
+  it("renders icon next to task status only if task is required", () => {
+    useMockRoadmap({
+      steps: [
+        generateStep({
+          section: "PLAN",
+          tasks: [generateTask({ required: true }), generateTask({ required: false })],
+        }),
+      ],
+    });
+
+    const subject = renderRoadmapPage();
+    const sectionPlan = subject.getByTestId("section-plan");
+    const allTasks = within(sectionPlan).getAllByRole("listitem");
+
+    expect(allTasks.length).toEqual(2);
+
+    expect(within(allTasks[0]).getByAltText(TaskDefaults.requiredIconAltText)).toBeVisible();
+    expect(within(allTasks[1]).queryByAltText(TaskDefaults.requiredIconAltText)).not.toBeVisible();
+  });
+
+  it("for a required task, hovering over icon renders tooltip", async () => {
```

Comment:
we like to keep the test names reading as sentences including the "it" (the reason is because it helps ensure that a test is describing the behavior of the component under test). So if we could rephrase this to read like a sentence where the component is the subject, that would be good. For example "it renders tooltip text when..."

Reply:
Oooh that makes sense